test/compute_bands.jl

```diff
@@ -99,7 +99,8 @@ end
 
 @testset "High-symmetry kpath construction for 1D system" begin
     lattice = diagm([8.0, 0, 0])
-    model   = Model(lattice; n_electrons=1, terms=[Kinetic()])
+    model   = Model(lattice; n_electrons=1, terms=[Kinetic()],
+                    disable_electrostatics_check=true)
```

Comment:
I think we need a special case for this type of situation.

We have a bunch of examples of this sort where there is not even electrostatics at all (e.g. first 1D examples, free electron examples). It does not make sense for people to need to explicitly disable this check here.